transport/http/outbound.go

```diff
@@ -399,6 +405,13 @@ func getYARPCErrorFromResponse(response *http.Response, bothResponseError bool)
 	)
 }
 
+func checkServiceMatch(reqSvcName string, resHeaders *http.Header) (bool, string) {
+	if resHeaders.Get(ServiceHeader) == "" || resHeaders.Get(ServiceHeader) == reqSvcName {
```

Comment:
nit: let's capture the service header in a variable instead of doing `resHeaders.Get(..)` in three places

Reply:
Resolved.